scripts/toolchain.sh

```diff
@@ -14,10 +14,18 @@ pushd "$(dirname "$0")/.."
 if which ccache >/dev/null 2>&1 && ([ -d 'run' ] && [ -w 'run' ] || [ -w '.' ]); then
     rm -rf 'run/toolchain'
     mkdir -p 'run/toolchain'
-    ln -sf "$(which ccache)" "run/toolchain/$CC"
-    ln -sf "$(which ccache)" "run/toolchain/$CXX"
-    export CC="$( realpath -e "run/toolchain")/$CC"
-    export CXX="$(realpath -e "run/toolchain")/$CXX"
+    export CR_CCACHE_CC_DIR="$( realpath -e "run/toolchain")"
+
+    for compiler in $CC $CXX; do
+        # Bazel may dereference the ccache symlink, so we use wrapper scripts instead.
+        # See https://github.com/bazelbuild/rules_cc/issues/130
+        cat << EOF > "$CR_CCACHE_CC_DIR/$compiler"
```

Comment:
You can indent EOF with underscores, and script with spaces.
```
cat <<< ____EOF | sed 's/^    //' > ...
    xxxxxxxx
____EOF
```

There's also a normal way to tell cat to ignore spaces before EOF, but I had some issue with that somewhere so rarely bother to try.